### PR TITLE
initial checkin

### PR DESCRIPTION
**Pull request by kdup**

### CODE REVIEW

**Comment by wvdixon** [comment]
This is a brilliant code change.
p1040612_view_bedroom_window

**Comment by hbushouse** [comment]
👍

File: trainees/hello_17.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello world!
```
Comment (nden) [inline trainees/hello_17.txt]: Hello

Comment (wvdixon) [inline trainees/hello_17.txt]: Consider adding a comma here.

**Pull request merged**